runtime/compiler/runtime/J9ValueProfiler.hpp

```diff
@@ -404,8 +424,28 @@ class TR_EmbeddedHashTable : public TR_HashTableProfilerInfo<T>
       }
 
    bool resetLowFreqKeys();
+#if defined(LINUX)
+#pragma GCC diagnostic push
+#pragma GCC diagnostic ignored "-Winvalid-offsetof"
+#elif defined(OSX) || (defined(AIXPPC) && JAVA_SPEC_VERSION >= 11)
+#pragma clang diagnostic push
+#pragma clang diagnostic ignored "-Winvalid-offsetof"
+#elif defined(AIXPPC) && JAVA_SPEC_VERSION < 11
```

Comment:
Linux BE build appeared missing (as mentioned by Peter).

Reply:
What does BE mean? How is it different from regular plinux builds?

Reply:
BE means Big Endian. We also have plinux Little Endian builds.
OpenJ9 only builds LE, IBM has both.

Reply:
it looks not right.  Linux BE build used the same (or similar) compiler as AIX java8 (instead of java11+).

Reply:
It should test the compiler levels and not java levels, as we should be free to change which compilers are used to compile the different versions without having to modify the code.

Reply:
agreed. you might be able to do that by testing predefined relevant macros existing or not, e.g. xlc and clang etc.

Reply:
Ok I think I did the control flow logic right this time, I'll run a build and see if the warnings go away